e2e_test/hawk_test.py

```diff
@@ -152,7 +156,10 @@ def main():
     browser.test('test_remove_clone', results, clone)
     browser.test('test_add_group', results, group)
     browser.test('test_remove_group', results, group)
-    browser.test('test_check_cluster_configuration', results, ssh)
+    if pacemaker_version < "2.1.8":
```

Comment:
it is OK for me, but notice that the comparison of version strings is different with string comarison

Reply:
Yes, thank you for the reminding!
I did some test as following, hope that can match the comparison
```
>>> "2.1.5+20221208.a3f44794f" < "2.1.8"
True
>>> "1.1.24+20210811.f5abda0ee" < "2.1.8"
True
>>> "2.1.7+20221208.a3f44794f" < "2.1.8"
True
>>> "2.1.70+20221208.a3f44794f" < "2.1.8"
True
>>> "2.1.8+20221208.a3f44794f" < "2.1.8"
False
```

Reply:
Some of the libraries are already using `LooseVersion` from `distutils.version`. You can perhaps use that one:
```
acarvajal@airfox:~/git/hawk/e2e_test [master|✔] > python3
Python 3.6.15 (default, Sep 23 2021, 15:41:43) [GCC] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from distutils.version import LooseVersion as Version
>>> Version("2.1.5+20221208.a3f44794f") < Version("2.1.8")
True
>>> Version("1.1.24+20210811.f5abda0ee") < Version("2.1.8")
True
>>> Version("2.1.7+20221208.a3f44794f") < Version("2.1.8")
True
>>> Version("2.1.70+20221208.a3f44794f") < Version("2.1.8")
False
>>> Version("2.1.8+20221208.a3f44794f") < Version("2.1.8")
False
```

Reply:
According to the document, I think the string comparison is acceptable for current situation. 
https://docs.python.org/3/tutorial/datastructures.html#comparing-sequences-and-other-types

Reply:
`rpm -q --queryformat '%{VERSION}'` only get the version string and not release string(such as `20221208.a3f44794f`), so the comparison is acceptable in most of case, but we need to avoid this case:
`2.1.8 > 2.1.10 is true`

Reply:
`rpm -q --queryformat '%{VERSION}'` only get the version string, we don't need to consider release string, the current comparison is acceptable in most cases(I checked the history of the version string, only included number and dot letters), but we also need to avoid the case like:
`2.1.8 > 2.1.10 is true`

Reply:
Good point @badboywj . Look above at the tests I did with `LooseVersion` that it correctly returns `False` when doing `2.1.70 < 2.1.8` :)

Reply:
Thanks Alvaro and Jun, I update commit to use the followed method to compare package version, how about you?
`zypper -t vcmp 2.1.8 $(rpm -q --queryformat '%{VERSION}' pacemaker)`

Reply:
Sounds great. Good finding.